agent/php_nrini.c

```diff
@@ -2009,6 +2061,18 @@ PHP_INI_ENTRY_EX("newrelic.high_security",
                  nr_high_security_mh,
                  0)
 
+PHP_INI_ENTRY_EX("newrelic.security.agent.enabled",
+                 "0",
+                 NR_PHP_SYSTEM,
+                 nr_security_agent_enabled_mh,
+                 0)
+
+PHP_INI_ENTRY_EX("newrelic.security.enabled",
+                 "0",
+                 NR_PHP_SYSTEM,
+                 nr_security_enabled_mh,
+                 0)
+
```

Comment:
Does the newrelic-php-agent need these settings? They don't seem to be used anywhere...

Reply:
newrelic.security.agent.enabled and newrelic.security.enabled settings are logged in the APM log file if these settings are disabled in the minit function

Reply:
Will the values of these settings be used by the security agent as well?

Reply:
Yes, these values are to be used by the security agent as well as mentioned [here](https://github.com/newrelic/newrelic-php-agent/pull/918#discussion_r1645443921) in comment.